app/views/teaching_events/show/_how-to-attend.html.erb

```diff
@@ -10,17 +10,24 @@
     <h2>How to attend</h2>
 
     <% if can_sign_up_online?(@event) %>
+
+      <% if @event.in_person? %>
       <p>
-        You must register for a place to attend this event.
+        Register for this event so you can check-in faster on arrival using a QR code, and sign-up for free email reminders about events near you.
```

Comment:
```suggestion
        Register for this event and we'll send you a QR code for a faster check-in when you arrive and free email reminders about events near you.
```
(sorry to be a pain) I'm just wondering if by saying 'sign up for free email reminders', it kind of sounds like we're implying that that's a separate process? i.e. register for this event, and then sign up for emails? (when I'm assuming what we mean is that if they sign up for the event, we'll automatically send them email reminders? they don't need to sign up again to something else?)

feel free to ignore if you disagree or if I've got this totally wrong!

Reply:
Should we say something like 'Register for this event **to secure your place**.....' - events do sometimes close early when full?

Agree with Ellie - not sure we need to talk about email reminders here - they may well get them already anyway through other sign-ups